data-models/README.md

```diff
@@ -2,4 +2,43 @@
 
 Common Akenza data models that are used throughout the device types.
 
-- Also refer to: https://github.com/smart-data-models/CrossSector and https://webthings.io/api/
+Data models are referenced by using the id and relevant definition:
+
+```
+{
+  "$id": "https://akenza.io/decentlab/iam/default.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "processingType": "uplink_decoder",
+  "topic": "default",
+  "title": "Default",
+  "properties": {
+    "temperature": {
+      "$ref": "https://akenza.github.io/device-types-library/data-models/ambiance/climate/schema.json#/$defs/temperatureC"
+    },
+    "humidity": {
+      "title": "Humidity",
+      "unit": "%",
+      "type": "number",
+      "description": "The relative humidity in %",
+      "minimum": 0,
+      "maximum": 100
```

Comment:
Also agree to set `minimum` and `maximum` here, as these values are known. Also agree to only introduce them, when their known to be a "well-known standard", as mentioned by you!